runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -584,6 +586,12 @@ private void hop(boolean previous)
 				}
 			}
 
+			// Avoid switching to near-max population worlds to prevent failure
+			if (world.getPlayers() >= (MAX_PLAYER_COUNT * MAX_HOP_TO_PERCENTAGE))
```

Comment:
FWIW, this would entirely prevent users from hopping to W2 or W330. I don't think that's desirable behavior.

Reply:
This code is in the hop up/down stuff. It has nothing to do with general hopping?

Reply:
ahh nice. Looks like I need to check more context before commenting, heh